api/aws/ec2/v1/ec2.proto

```diff
@@ -67,7 +68,7 @@ message ResizeAutoscalingGroupResponse {
 message AutoscalingGroup {
   option (clutch.api.v1.id).patterns = {
     type_url : "clutch.aws.ec2.v1.AutoscalingGroup",
-    pattern : "{region}/{name}"
+    pattern : "{account}/{region}/{name}"
```

Comment:
the one issue i think i've found is that a query like us-east-1/foo-bar will no longer limit the query to us-east-1

Reply:
🤔 let me see what i can do

Reply:
I would think in a case like this I would ideally like to let the fanout to finish instead of short circuiting once it finds a single result to try and capture if there was more than one resource found with the same name and propagate up an error if so, instead of returning the first one we find.

I think this would be a more fundamental change as I would think this should apply to any resolver